Framework/CMakeLists.txt

```diff
@@ -424,7 +424,8 @@ install(FILES example-default.json
         DESTINATION etc)
 
 install(PROGRAMS script/RepoCleaner/o2-qc-repo-cleaner
-                 script/RepoCleaner/o2-qc-repo-deletion
+                 script/RepoCleaner/o2-qc-repo-delete-runs
+                 script/RepoCleaner/o2-qc-repo-delete-period
```

Comment:
I was thinking that you mean data taking period (as the metadata `PeriodName`). Could you please clarify this in the executable name? E.g.  `o2-qc-repo-delete-interval`, o2-qc-repo-delete-time-period`?